dds/src/dcps/infrastructure/qos.rs

```diff
@@ -39,6 +40,9 @@ pub struct DomainParticipantQos {
     pub user_data: UserDataQosPolicy,
     /// Value of the entity factory QoS policy.
     pub entity_factory: EntityFactoryQosPolicy,
+    /// Value of the discovery config QoS policy (vendor extension).

```

Comment:
The QoS is strictly defined in the DDS standard and the lease duration is something coming from RTPS so this would need to go back to configuration instead of QoS.